src/fmu/settings/models/user_info.py

```diff
@@ -0,0 +1,15 @@
+"""Model for the log entries in the the userlog file."""
+
+from datetime import UTC, datetime
+
+from pydantic import AwareDatetime, BaseModel, ConfigDict, Field
+
+
+class UserInfo(BaseModel):
```

Comment:
We can start with this. I think this model is generic enough to serve as the base for any event/structured log that allows arbitrary extras. No idea what a great name for that is.